app/views/layouts/listnav/_ems_storage.html.haml

```diff
@@ -6,13 +6,13 @@
     = miq_accordion_panel(_("Properties"), false, "ems_prop") do
       %ul.nav.nav-pills.nav-stacked
         = li_link(:if => true,
-          :text       =>_('Summary'),
+          :text       => _('Summary'),
           :record_id  => @record.id,
           :display    => 'main',
           :title      => _("Show Summary"))
-        = li_link(:if => (@record.has_events? || @record.has_events?(:policy_events)),
```

Comment:
@h-kataria Does it mean that Timelines is not broken from accordion on Storage Managers summary page but the bug is that it is enabled in Monitoring tab even there is no timeline data?